plugin/pom.xml

```diff
@@ -169,6 +169,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.8.4</version>
```

Comment:
This dependency is probably added to compile the sample.
To keep the project light and free of version conflict, we are using mock class and interface ([See plugin-dep](https://github.com/find-sec-bugs/find-sec-bugs/tree/master/plugin-deps/src/main/java)). The sample are not executed anyway.

In your case, you would need to create the stub classes for [com.fasterxml.jackson.annotation.JsonTypeInfo and 
com.fasterxml.jackson.databind.ObjectMapper](
https://github.com/find-sec-bugs/find-sec-bugs/pull/306/commits/a91cdf640dad328e167a7166007e4a536667be3f#diff-868763a9b824d007c96a6db59d31b4a2R3).